Dockerfile.ocp

```diff
@@ -1,4 +1,5 @@
-FROM ironic-builder AS builder
+ARG BaseImage
+FROM ${BaseImage:-"base-os"} AS builder
```

Comment:
shouldn't the default be `ironic-builder`? or do you mean to change it?

Reply:
yes, it was a test, I moved it back to ironic-builder